libcxx/include/deque

```diff
@@ -61,7 +61,7 @@ public:
     deque& operator=(deque&& c)
         noexcept((__alloc_traits::propagate_on_container_move_assignment::value &&
                   is_nothrow_move_assignable<allocator_type>::value) ||
-                 allocator_traits<allocator_type>::is_always_equal::value);
+                 __alloc_traits::is_always_equal::value);
```

Comment:
I would actually keep using `allocator_traits<allocator_type>` in the synopsis (but I would do so consistently) since `__alloc_traits` is not defined in the synopsis. So I'd change to

```
 noexcept((allocator_traits<allocator_type>::propagate_on_container_move_assignment::value &&
                   is_nothrow_move_assignable<allocator_type>::value) ||
                  allocator_traits<allocator_type>::is_always_equal::value);
```

However, in the code below I'd use `__alloc_traits` consistently.